### PR TITLE
Remove license comment for azurewebsites.net or umbraco.io

### PR DESCRIPTION
**Pull request by mattbrailsford**
Umbraco Commerce doesn't support azurewebsites.net or umbraco.io as dev domains OOTB, these now need to be requested.

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @mattbrailsford ! 💪 

Do we have any docs on how to request these, then? 🤔 
And how about the Cloud domains - do we need some sort of guide for getting it to work on Cloud?

**Comment by sofietoft** [comment]
Yeah, that could work.
Added as a little hint or something!

**Comment by mattbrailsford** [comment]
Ok, I've updated the last sentence in that block to include reaching out to suits

File: 12/umbraco-commerce/getting-started/installation.md [hunk 1]
```diff
@@ -57,14 +57,12 @@ You only need to install your license when you are ready to go live.
...
-
 Hosting on any other public domain without a license will be limited to a maximum of 20 orders.
 
 If you require an unrestricted staging environment, all licenses support two methods of allowing this:
 
 * `staging.client.com` - Licenses allow unlimited subdomains, meaning you can host the staging site on a subdomain of the licensed domain.
 * `clientcom.agency.com` - Licenses allow a concatenation of the licensed domain as a subdomain of any other domain.
 
-If you wish to host the site on any other URL, then an additional license file will be required for that domain.
+If you wish to host the site on any other URL such as a public dev / staging domain you want to access, please reach out to suits@umbraco.com for a test license.
```
Comment (sofietoft) [inline 12/umbraco-commerce/getting-started/installation.md:67]: Shortened the sentence a bit.

```suggestion
If you wish to host the site on any other URL like a public development domain, reach out to suits@umbraco.com for a test license.
```

**Review by sofietoft — COMMENTED** [review]
Ah! Hadn't seen this was already part of a hint - my bad!

I've made a single suggestion to shorten the sentence a bit.

**Comment by mattbrailsford** [comment]
Yup, agree that's shorter and cleaner. All good from my end 👍

**Comment by sofietoft** [comment]
Excellent. I'll merge it.

This should be added to the 10 version as well, correct?

**Comment by mattbrailsford** [comment]
It should yes 👍

**Comment by sofietoft** [comment]
Super. I'll take care of that after the fact.

**Pull request merged**